stl/inc/tuple

```diff
@@ -179,19 +179,51 @@ struct _Alloc_unpack_tuple_t {
     explicit _Alloc_unpack_tuple_t() = default;
 }; // tag type to disambiguate construction (from an allocator and unpacking a tuple/pair)
 
+#if _HAS_CXX23 && defined(__cpp_lib_concepts)
+template <class _Other, class _Tuple, class _Indices = make_index_sequence<tuple_size_v<_Tuple>>>
+struct _Can_construct_values_from_tuple_like;
+
+template <class _Other, class... _Types, size_t... _Indices>
+struct _Can_construct_values_from_tuple_like<_Other, tuple<_Types...>, index_sequence<_Indices...>>
+    : conjunction<is_constructible<_Types, decltype(_STD get<_Indices>(_STD declval<_Other>()))>...> {};

```

Comment:
I guess this should be turned into a variable as it is not furtherly used within `conjunction(_v)`/`disjunction(_v)` (ditto for `_Is_tuple_like_constructor_explicit` and `_Can_assign_values_from_tuple_like`).

Reply:
Done